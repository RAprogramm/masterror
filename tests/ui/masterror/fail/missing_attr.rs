use masterror::Masterror;

#[derive(Debug, Masterror)]
#[error("no attribute")]
struct Missing;

fn main() {}
