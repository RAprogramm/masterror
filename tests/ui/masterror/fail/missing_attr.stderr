error: #[derive(Masterror)] requires #[masterror(...)] on structs
 --> tests/ui/masterror/fail/missing_attr.rs:9:8
  |
9 | struct Missing;
  |        ^^^^^^^
